frontstage/views/sign_in/sign_in.py

```diff
@@ -111,3 +118,13 @@ def resend_verification_expired_token(token):
     party_controller.resend_verification_email_expired_token(token)
     logger.info('Re-sent verification email for expired token.', token=token)
     return render_template('sign-in/sign-in.verification-email-sent.html')
+
+
+def obfuscate_email(email):
```

Comment:
Is there a unit test for this?

Reply:
Good spot.  There isn't and is a perfect candidate for one!